extension/src/services/binding.ts

```diff
@@ -183,14 +197,36 @@ export default class Binding {
                 };
                 this.subtitleController.notification('info.enabledCondensedPlayback');
                 break;
+            case PlayMode.fastForward:
+                if (this._playMode === PlayMode.condensed) {
+                    disableCondensedMode();
+                }
+                this.subtitleController.fastForwardBetweenSubtitles = async (slice) => {
```

Comment:
It would be more consistent with the naming of the other hooks to call this something like `onSlice` which isn't specific to fast forwarding.